src/main/java/forceitembattle/listener/Listeners.java

```diff
@@ -145,10 +243,61 @@ public void onClose(InventoryCloseEvent inventoryCloseEvent) {
 
     @EventHandler
     public void onEntityDamage(EntityDamageEvent event) {
-        if (ForceItemBattle.getTimer().isRunning()) {
-            if (!(event.getEntity() instanceof Player)) return;
-            event.setCancelled(false);
-        } else event.setCancelled(true);
+        if (!ForceItemBattle.getTimer().isRunning()) {
+            event.setCancelled(true);
+        }
+
+        if (isPvpEnabled() || !(event.getEntity() instanceof Player)) {
+            return;
+        }
+
+        // Disable Fire damage if pvp disabled and there's another player nearby
+        if (event.getCause() == EntityDamageEvent.DamageCause.FIRE || event.getCause() == EntityDamageEvent.DamageCause.FIRE_TICK || event.getCause() == EntityDamageEvent.DamageCause.LAVA) {

```

Comment:
this means just light someone on fire or drop them into lava and then walk away 5 blocks... Better to actually track from which block the player got burnt and like have a set of blocks that all other players lit on fire. For spleefing someone into lava there isn't really a fix I guess